public/electron.js

```diff
@@ -1,56 +1,34 @@
 const {app, BrowserWindow} = require('electron');
-const path = require('path');
-const url = require('url');
-
-// Keep a global reference of the window object, if you don't, the window will
-// be closed automatically when the JavaScript object is garbage collected.
-let win;
-
+const {join} = require('path');
+const {format} = require('url');
+let isDev = process.env.NODE_ENV === 'development';
+let mainWindow;
 function createWindow() {
-    // Create the browser window.
-    win = new BrowserWindow({width: 800, height: 600});
-
-    // Url is either set using environment variable (see .env, used for dev) or index.html file (for prod)
-    const startUrl = process.env.ELECTRON_START_URL || url.format({
-        pathname: path.join(__dirname, '../build/index.html'),
-        protocol: 'file:',
-        slashes: true
-    });
-
-    // Load start url of the app.
-    win.loadURL(startUrl);
-
-    // Open the DevTools.
-    //win.webContents.openDevTools()
-
-    // Emitted when the window is closed.
-    win.on('closed', () => {
-        // Dereference the window object, usually you would store windows
-        // in an array if your app supports multi windows, this is the time
-        // when you should delete the corresponding element.
-        win = null;
-    })
+    mainWindow = isDev ? new BrowserWindow({
+        width: 900,
+        height: 680,
+    }) : new BrowserWindow({ width: 900, height: 680 });
+    const startUrl = isDev
+        ? 'http://localhost:3000/?electron=true'
+        : format({
+            pathname: join(__dirname, '/../build/index.html'),
```

Comment:
Let's pass `?electron=true` for `/../build/index.html` as well, this will simplify check in App code.